manifests/0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml

```diff
@@ -102,6 +102,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: OVN_EGRESSIP_HEALTHCHECK_PORT
```

Comment:
oops, this was not supposed to be here....

Reply:
Update: This oops has been fixed here: https://github.com/openshift/cluster-network-operator/pull/1568